tests/src/JIT/SIMD/CircleInConvex.cs

```diff
@@ -259,8 +259,9 @@ static int Main(string[] args)
             float r;
             FindCircle(points, out O, out r);
 
-            float expRes = 7.565624E7F;
-            if (Math.Abs(r - expRes) > EPS)
+            float expRes = 75656240.0F;
+            float ulp    =        8.0F;
```

Comment:
I'm unsure how the value 8.0F was determined, but I do not that 
 EPS * expRes * steps = 7.5656, using 1E-09 for EPS